### PR TITLE
new module: pyrodigal

### PR DESCRIPTION
**Pull request by louperelo**
Closes issue #3114 

## PR checklist

Closes #XXX 

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ x] Remove all TODO statements.
- [x ] Emit the `versions.yml` file.
- [x ] Follow the naming conventions.
- [x ] Follow the parameters requirements.
- [ x] Follow the input/output options guidelines.
- [ x] Add a resource `label`
- [x ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ x] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/pyrodigal/meta.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+name: "pyrodigal"
+description: write your description here
```
Comment (SusiJo) [inline modules/nf-core/pyrodigal/meta.yml:2]: Can you please add a description here?

Comment (SusiJo) [inline modules/nf-core/pyrodigal/meta.yml:2]: Can you please add some short description here?

File: tests/modules/nf-core/pyrodigal/main.nf [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+workflow test_pyrodigal {
+
+    input = [
+        [ id:'test', single_end:false ], // meta map
+        file(params.test_data['haemophilus_influenzae']['genome']['genome_fna_gz'], checkIfExists: true)
+    ]
+
+    GUNZIP( input )
+    GUNZIP.out.gunzip.view()
```
Comment (SusiJo) [inline tests/modules/nf-core/pyrodigal/main.nf:16]:
```suggestion
```

**Comment by louperelo** [comment]
@nf-core-bot fix linting

**Review by jasmezz — CHANGES_REQUESTED** [review]

**Review by jasmezz — APPROVED** [review]

**Review by SusiJo — APPROVED** [review]
LGTM! 🚀

**Pull request merged**